tests/stub/driver_execute_query/scripts/tx_return_1_with_auth.script

```diff
@@ -0,0 +1,18 @@
+!: BOLT 5.1
+
+A: HELLO {"user_agent": "*", "[routing]": "*"}
```

Comment:
```suggestion
A: HELLO {"{}": "*"}
```
I guess we don't really care about any of these fields in this test, do we? Or are we interested in asserting that certain fields are not contained in the HELLO request? I feel like other tests should probably cover this... Not 100% sure.

I won't comment again on the other new script, but the same discussion applies.

Reply:
Yeah agree this can be tested in other tests 👍